e2e/cache-client.e2e.ts

```diff
@@ -18,7 +18,22 @@ describe.skip('Cache client', () => {
     );
     const { connectToDidRegistry, assetsService } =
       await connectToCacheServer();
-    const { claimsService, didRegistry } = await connectToDidRegistry();
+    const projectId = '2DCe3TNAHzzmoSBrcPyWx4NOyrv';
```

Comment:
I will change these configs - what configs should we be using for test files? I will make helper and share among e2e tests.

Reply:
Best to use
ProjectID: 2DD5GcJRpMeuqMoKsgT95AQ8OI3
ProjectSecret: 842995992aa1edd763196b30d727a45d